solutions/silver/cf-1732C1.mdx

```diff
@@ -0,0 +1,159 @@
+---
+id: cf-1732C1
+source: CF
+title: "Sheikh (Easy version)"
+author: Ryan Chou
+---
+
+[Official Editorial](https://codeforces.com/blog/entry/108327)
+
+<Spoiler title="Hint 1">
+What's the maximum value of $f(l, r) = \operatorname{sum}(l, r) - \operatorname{xor}(l, r)$?
+</Spoiler>
+
+<Spoiler title="Hint 1 (answer)">
+Note that the XOR of two numbers can never grow _more_ than their sum.
+
+Thus, the maximum value of $f(l, r)$ is $f(1, n)$.
+We've now reduced the problem to finding the minimum-length subsegment that achieves this maximum!
+</Spoiler>
+
+<Spoiler title="Solution">
+## Explanation
+
+From the hint, we've now just have to find the minimum-length subsegment that achieves the maximum value of $f(l, r)$.
+
+Remember that since the XOR of two numbers can never grow more than their sum, this length is actually monotonically increasing<Asterisk>If $x$ works, then $x + 1$ will always work.</Asterisk>!
```

Comment:
explain this more. How do you go from "since the XOR of two numbers can never grow more than their sum" to "this length is actually monotonically increasing"